examples/08 Introduction to PVMT.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Property Value Management (PVMT)\n",
+    "\n",
+    "This notebook present the current capabilities of the PVMT extension for py-capellambse.\n",
+    "We will see how to display all defined property value domains, their groups and the\n",
+    "available properties.\n",
+    "Further we'll see how to access applied property values from ModelObjects. This is demonstrated on the list of LogicalActors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import capellambse\n",
+    "\n",
+    "\n",
+    "path_to_model = \"../tests/data/melodymodel/5_2/Melody Model Test.aird\"\n",
+    "model = capellambse.MelodyModel(path_to_model, jupyter_untrusted=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can investigate all defined PVMT domains, their groups and properties:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<pvmt.Domain \"DarkMagic\"(3763dd54-a878-446a-9330-b5d9c7121865)>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<pvmt.Group \"Power\"(1e241678-282c-46f3-abbd-e0cf34e743ee)>,\n",
+       " <pvmt.Group \"Power Level\"(a0cb5a23-955e-43d6-a633-2c4e66991364)>]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<pvmt.FloatPropertyValue \"Min\"(4c5eba40-fcc3-480f-b88f-e375dadc00e5)>,\n",
+       " <pvmt.FloatPropertyValue \"Max\"(196467a2-ecee-4f35-8987-6a898cd34616)>]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<pvmt.EnumerationPropertyValue \"Skill\"(7367c731-aabb-4807-9d74-7ef2c6482ec9)>]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "darkmagic_domain = model.pvext.domains[0]\n",
+    "display(darkmagic_domain)\n",
+    "display(darkmagic_domain.groups)\n",
+    "for grp in darkmagic_domain.groups:\n",
+    "    display(grp.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The domain scope (i.e. what ArchitectureLayer) this extension can be apploed to is not yet processed further. Scopes have rules underneath to control on which ModelObject's this can be applied to.\n",
```

Comment:
```suggestion
    "The domain scope (i.e. what ArchitectureLayer) this extension can be applied to is not yet processed further. Scopes have rules underneath to control on which ModelObject's this can be applied to.\n",
```